src/blueapi/BlueapiComponents.tsx

```diff
@@ -47,6 +48,10 @@ export function RunPlanButton(props: RunPlanButtonProps) {
   const params = props.planParams ? props.planParams : {};
   const variant = props.btnVariant ? props.btnVariant : "outlined";
   const size = props.btnSize ? props.btnSize : "medium";
+  const color = props.btnColor ? props.btnColor : "primary";
```

Comment:
Could: go back to "custom" as default so the start and abort run buttons all have the same look